### PR TITLE
Screenshots

### PR DESCRIPTION
**Pull request by khayitmuradov**
added CustomLogger
added Screenshot

### CODE REVIEW

File: UI/Utils/WebDriverUtils.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UI.Utils
+{
```
Comment (NikitaGavruk) [inline UI/Utils/WebDriverUtils.cs]: Delete this file

**Review by NikitaGavruk — CHANGES_REQUESTED** [review]
As I see, Logger.cs and CustomLogger.cs both make logger job. Please leave only one from them. I prefer the CustomLogger.cs

File: UI/Utils/WebDriverUtils.cs (hunk 1 above)
Comment (khayitmuradov) [reply UI/Utils/WebDriverUtils.cs]: deleted

**Comment by khayitmuradov** [comment]
deleted Logger.cs, WebDriverUtils and Screenshots.cs files

**Pull request merged**